skrf/tests/test_vectorfitting.py

```diff
@@ -1,40 +1,50 @@
 import unittest
-
 import pytest
 import skrf
 import numpy as np
 import tempfile
 import os
 import sys
-import warnings
 
 
 class VectorFittingTestCase(unittest.TestCase):
 
+    msg_passivity_violation = 'The fitted network is passive, but the vector fit is not passive. Consider running ' \
+                              '`passivity_enforce()` to enforce passivity before using this model.'
+
+    @pytest.mark.filterwarnings(f'ignore:*{msg_passivity_violation}*:UserWarning')
```

Comment:
Have you tried without the `*` character?

Reply:
It wasn't the `*`, but it now works in general form, without any message. That filters every UserWarning, which I think is fine in this case.